src/main/resources/assets/locks/lang/pl_pl.json

```diff
@@ -1,26 +1,26 @@
 {
-	"itemGroup.locks": "Zamki",
+	"itemGroup.locks": "Locks",
```

Comment:
"Locks" here is a proper noun, so it shouldn't be translated. Don't you think that the best idea would be to hardcode this string and remove it from all translation files?

Reply:
I feel like it was fine before. Should be translated

Reply:
If I remember correctly, mods usually use their name as a creative tab name, and in this case the creative tab name is a proper noun, so it shouldn't be translated. On the other hand, large mods sometimes put their name and an additional word or two (e.g. Industrial Foregoing: Tools, Industrial Foregoing: Transport), and in this case the first part is still a proper noun, but the second one is not, so of course it should be translated.

Reply:
I'm not sure. Looking at other bigger mods, I see that their names are translated (e.g. botania)

Reply:
Botania adds only one creative tab and uses its name as the tab name. Some translators unnecessarily translated it (perhaps because they didn't realize that it is a proper noun, so it shouldn't be translated), but the majority did not.